src/Umbraco.Cms.Api.Management/Factories/AuditLogPresentationFactory.cs

```diff
@@ -19,19 +20,17 @@ public AuditLogPresentationFactory(IUserService userService, IUserIdKeyResolver
 
     public IEnumerable<AuditLogResponseModel> CreateAuditLogViewModel(IEnumerable<IAuditItem> auditItems) => auditItems.Select(CreateAuditLogViewModel);
 
-    private AuditLogResponseModel CreateAuditLogViewModel(IAuditItem auditItem)
-    {
-        Guid userKey = _userIdKeyResolver.GetAsync(auditItem.UserId).GetAwaiter().GetResult();
-        IUser user = _userService.GetAsync(userKey).GetAwaiter().GetResult()
```

Comment:
As the user service does no extra validations when retrieving the user, I removed this call entirely, as it seemed unnecessary as we only need the key.